### PR TITLE
446 participants list and evaluation view

### PR DESCRIPTION
**Pull request by ferrlin**
ready for review and testing

### CODE REVIEW

File: app/assets/element/participants.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+$(document).ready(function(){
+
+	$('body').append(
+		$('<div id="confirmEmail" class="modal fade" tabIndex="-1">').
+        attr('id', dialog).attr('role', 'dialog').
+        attr('aria-hidden', 'true').append(html));
+
+	$('#confirmLink').on('click', function(e){
+		var ids = $.data(this, 'ids');
```
Comment (sery0ga) [inline app/assets/element/participants.js]: it's not good. What's the reason to send multiple server requests when we could do one and everything will happen on a server side? It's more user friendly, faster and less error-prone

Comment (ferrlin) [inline app/assets/element/participants.js]: Good point. I'll update it.

File: app/assets/element/participants.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	    	$('#confirmEmail').modal('show');
+	    }, delay);
+	} else console.log("No participants to resend email.");
+}
+
+$(document).ready(function(){
+
+	$('body').append(
+		$('<div id="confirmEmail" class="modal fade" tabIndex="-1">').
+        attr('id', dialog).attr('role', 'dialog').
```
Comment (sery0ga) [inline app/assets/element/participants.js]: This is unclear to me. Why can't you use pure html code, without javascript generation of <div id="confirmEmail">?

File: app/assets/element/participants.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+			var response = JSON.parse(data);
+			success(response.message);
+		})
+		.fail(function(data){
+			var response = JSON.parse(data);
+			fail(response.message);
+		});
+
+		$('#' + dialog).modal('hide');
+		$('#confirmEmail').modal('hide');
```
Comment (sery0ga) [inline app/assets/element/participants.js]: This also look strange. Bootstrap modals could be shown one at a time. Why do you expect to have two modals open?

File: app/assets/element/participants.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+ * continue resending emails to participants
+ */
+function resendEmailsToAll(data) {
+	if(data.length > 0){
+		$('#confirmLink').data({'ids': data});
+	    $('#' + dialog).modal('show');
+	    window.setTimeout(function(){
+	    	$('#confirmEmail').modal('show');
+	    }, delay);
+	} else console.log("No participants to resend email.");
```
Comment (sery0ga) [inline app/assets/element/participants.js]: At least say something to user. Right now the behaviour is not misguiding - the user clicks on the link and sees nothing

File: app/assets/element/participants.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+/**
+ * continue resending emails to participants
+ */
+function resendEmailsToAll(data) {
+	if(data.length > 0){
+		$('#confirmLink').data({'ids': data});
+	    $('#' + dialog).modal('show');
+	    window.setTimeout(function(){
+	    	$('#confirmEmail').modal('show');
+	    }, delay);
```
Comment (sery0ga) [inline app/assets/element/participants.js]: 25 milliseconds is very small timeout. From user perspective there's no timeout at all. Why do you want to show "resend" dialog at all? May be I don't understand some logic here.

**Pull request closed without merging**